send-transaction-service/src/transaction_client.rs

```diff
@@ -20,8 +22,10 @@ use {
     },
     tokio::{
         runtime::Handle,
-        sync::mpsc::{self},
-        task::JoinHandle as TokioJoinHandle,
+        sync::{
+            mpsc::{self},
+            watch,
+        },
```

Comment:
&lt;minor&gt;
It is strange fmt did not reformat this part.
My understanding is that it should be like this:

```suggestion
        sync::{mpsc, watch},
```